### DISCUSSION TITLE
Experiencia técnica

### DISCUSSION THREAD

**Discussion by Topi99** [Solify-IT/docs#1] [Ideas]
Comenten aquí su experiencia técnica. Cualquier lenguaje/herramienta es bienvenida.

**Comment by Topi99** [comment]
Yo tengo experiencia con:

  * JavaScript/TypeScript (React, Vue, Node, Express).
  * Python (Django, Flask, Falcon).
  * Go (Apenas estoy aprendiendo, estoy tomando un Bootcamp, pero estaría bien usarlo).

**Comment by scarlosro** [comment]
Mi experiencia es:

* JavaScript (React, Node, Express) Básico
* Python (Django)

**Comment by albertcastaned** [comment]
- Django + Django REST API
- Flask + Flask REST + Marshmellow
- JavaScript (Un poco de React)
- PHP puro :C

**Comment by domerent97** [comment]
Yo tengo experiencia en 

- TypeScript (Angular, Ionic)
- Python, pero no lo he usado con un framework

**Comment by BurksonB** [comment]
Tengo experiencia en:

- Python (Django, Flask)
- Javascript (React, un poco de express)

**Comment by LaloGlzMlz** [comment]
Python (Django, Flask)
JavaScript (Poco en react y JavaScript puro)

**Comment by AdrizPaola** [comment]
- JavaScript (Node, Express, React)
- TypeScript (Nest)

**Comment by ricardo-esc** [comment]
- JavaScript (Node, Express, React, Mongoose)
- NetCore 3.0

**Comment by arcosmarce** [comment]
Javascript (React)
Typescript( un poco de angular)

**Comment by KevinWoolfolk** [comment]
Yo tengo experiencia con:
Php(Mysql, Laravel, Html, Css y Js).
Flutter(Dart, Firebase).

**Comment by JuanCarlosCV** [comment]
Experiencia con:
Php,Html,Laravel, Css, java, .net, python
JavaScript(React)
TypeScript,Angular
Bases de datos( mysql,access,sqlserver)
